src/Form/ConfigSettingsForm.php

```diff
@@ -60,6 +60,18 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#description' => $this->t('The default tax code to send to TaxCloud when calculating taxes, if company code is not set on the purchased entity of a given order item.'),
     ];
 
+    $form['configuration']['tax_rounding_method'] = [
+      '#type' => 'radios',
+      '#title' => t('Rounding method'),
+      '#default_value' => $config->get('tax_rounding_method') ?: '',
+      '#options' => [
+        'per_line' => 'Round per Line',
+        'globally' => 'Round Globally',
+      ],
+      '#required' => TRUE,
+      '#description' => $this->t('How total tax amount is computed in orders.'),
```

Comment:
I think we need to make a decision what to use everywhere in this form:
$this->t() or t(), but an approach should use only one method.